src/components/input-field/input-field.tsx

```diff
@@ -345,6 +345,14 @@ export class InputField {
             return;
         }
 
+        if (
+            this.type === 'number' &&
+            this.isFocused &&
+            Number(newValue) === Number(this.mdcTextField.value)
+        ) {
+            return;
```

Comment:
when I test this, I see the exact same issue as described in the issue itself https://github.com/Lundalogik/lime-elements/issues/1402

Reply:
Sorry, I think this PR depends on https://github.com/Lundalogik/lime-elements/pull/2869

There's lots of small fixes in there.